feature-libs/product-configurator/rulebased/components/overview-form/configurator-overview-form.component.html

```diff
@@ -24,61 +24,119 @@ <h2>{{ 'configurator.overviewForm.noAttributeHeader' | cxTranslate }}</h2>
   <span class="cx-visually-hidden">
     {{ 'configurator.a11y.listOfAttributesAndValues' | cxTranslate }}
   </span>
-  <ng-container *ngFor="let group of overviewGroups">
```

Comment:
There is a number of changes in HTML tree. 
Is there any task to cover them, for instance see `5_0.md`?